### ISSUE TITLE
Bootstrap 4 wrapper does not fire collapse event

### ISSUE THREAD

**Issue by abombelli** [FrDH/jQuery.mmenu#889]
Hello

We use the Bootstrap 4 wrapper and use the toggler button.
The menu opens fine when clicking the toggler through the wrapper, but when clicking the icon again to close nothing happens.

I tried this simple code to check if the click event is fired, but nothing happens.

```
$("#navbar-toggler-button").click(function() {
  alert('toggler clicked');
});
```

So the wrapper does somehow not create a close event.

Any idea?

Thanks,
Aldo

**Comment by FrDH** [comment]
Hi,

I guess the assumption was the toggler would be overlaped by the "page-blocker" that always closes the menu when clicked.

I see the wrapper always opens the menu [when the toggler is clicked](https://github.com/FrDH/jQuery.mmenu/blob/master/src/wrappers/bootstrap/jquery.mmenu.bootstrap4.ts#L77).

I guess that could use an `if/else` in there.
I'll put that on the TODO list 👍

**Comment by abombelli** [comment]
Great!
Is there a hack do the meantime?

**Comment by abombelli** [comment]
Hello,
Any news on this?
Or maybe a hack for the time being?

**Comment by FrDH** [comment]
Should be fixed in 7.0.6
